### PR TITLE
Rename point light type to omni, add shapes and shape picture

### PR DESCRIPTION
**Pull request by raytranuk**
image

### CODE REVIEW

File: content/en/user-manual/graphics/lighting/lights.md [hunk 1]
```diff
@@ -26,27 +28,23 @@ A directional light lights an object like this:
...
+An omni light lights an object like this:
 
-Point lights are more costly than directional lights but faster to process than spot lights.
+![Omni light][4]
 
-## Spot Lights
+### Spot Lights
 
-Spot lights, like point lights, emit light from a single point in space. However, the light from the spot light is constrained to a cone shape. Examples of a spot light would be a torch or the light from a lighthouse.
+Spot lights, like point lights, emit light from a single point in space. However, the light from the spot light is constrained to a cone shape. Examples of a spot light are provided in the [table](#use-cases) below.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/lighting/lights.md:47]:
```suggestion
Spot lights, like omni lights, emit light from a single point in space. However, the light from the spot light is constrained to a cone shape. Examples of a spot light are provided in the [table](#use-cases) below.
```

File: content/en/user-manual/graphics/lighting/lights.md [hunk 2]
```diff
@@ -58,11 +56,53 @@ A spot light lights an object like this:
...
+There are four light source shapes:
+
+* Punctual
+* Rectangle
+* Disk
+* Sphere
+
+### Punctual
+
+The punctual light source shape is an infinitesimally small point. This is the default light source shape and is a less physically correct, but relatively low cost approximation of a light source. The other light source shapes are more costly to render but will give more correct ligthing and specular reflections. Examples given in [table](#use-cases) below.
```
Comment (willeastcott) [inline content/en/user-manual/graphics/lighting/lights.md:70]: This page uses the phrase `Examples given in [table](#use-cases) below.` several times. I'm not sure this is really necessary.

File: content/en/user-manual/graphics/lighting/lights.md (hunk 1 above)
Comment (raytranuk) [reply content/en/user-manual/graphics/lighting/lights.md]: Thanks!

File: content/en/user-manual/graphics/lighting/lights.md (hunk 2 above)
Comment (raytranuk) [reply content/en/user-manual/graphics/lighting/lights.md:70]: Good point - I will rewrite this to avoid repetition

**Review by willeastcott — APPROVED** [review]

File: content/en/user-manual/graphics/lighting/lights.md (hunk 2 above)
Comment (raytranuk) [reply content/en/user-manual/graphics/lighting/lights.md]: done

**Pull request merged**